scout/server/blueprints/institutes/templates/overview/institute_settings.html

```diff
@@ -104,6 +104,13 @@
       placeholder: "Add Sanger email",
     });
 
+    $('#soft_filters').select2({
+      tags: true,
+      theme: 'bootstrap-5',
+      tokenSeparators: [','],
+      placeholder: "tnscope:germline_risk",
```

Comment:
Wasn't it only 
```suggestion
      placeholder: "germline_risk",
```

Reply:
Ah yes, that was a previous (now outdated) version. Thanks!